### PR TITLE
Pre release stuff

### PR DESCRIPTION
**Pull request by wxtim**
- [x] Double checked `setup.py` for correct version of cylc-flow (8.0a1).
- [x] Changed the version number in `metomi/rose/__init__.py`
- [x] Added some release notes.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -5,6 +5,27 @@ for a full listing of issues for each release.
 
 --------------------------------------------------------------------------------
 
+## 2.0a1 (2019-09-19)
+
+Rose release 59. This maintenance release is expected to be used with:
```
Comment (matthewrmshin) [inline CHANGES.md]: Should replace this with info that this is a pre-release and is not suitable for normal use.

File: CHANGES.md [hunk 2]
```diff
@@ -5,6 +5,27 @@ for a full listing of issues for each release.
...
+
+[#2288](https://github.com/metomi/rose/pull/2288):
+Rosie & Rosa: migrate to Python 3(.6-.7) & Tornado
+
+[#2317](https://github.com/metomi/rose/pull/2317):
+Python3 Conversion of Rose Suite Run
+
+[#2366](https://github.com/metomi/rose/pull/2366):
+Rose and Rosie: Make Rosie Work under packaging
+
```
Comment (matthewrmshin) [inline CHANGES.md]: Should add to the fact that we have deleted the GUIs.

Comment (matthewrmshin) [reply CHANGES.md]: And that Rose will now be deployed to PyPI and will be expected to be installed via Conda.

**Comment by wxtim** [comment]
Quite right @matthewrmshin - I've re-written it a fair bit, hopefully to your liking.

**Comment by matthewrmshin** [comment]
Not part of this change, but it looks like we need to give Rose test battery a bit of TLC.

**Comment by matthewrmshin** [comment]
Travis CI is happy! 😄

**Pull request merged**